pyaedt/modeler/modeler3d.py

```diff
@@ -127,18 +127,21 @@ def create_3dcomponent(
             Component outline. Value can either be ``BoundingBox`` or ``None``.
             The default is ``BoundingBox``.
         auxiliary_dict : bool or str, optional
-            Whether to export the auxiliary file containing information about defined datasets and Icepak monitor
-            objects. A destination file can be specified using a string.
+            Whether to export the auxiliary file containing information about defined
+            datasets and Icepak monitor objects. A destination file can be specified
+            using a string.
             The default is ``False``.
         monitor_objects : list, optional
-            List of monitor objects' names to export. The default is the names of all monitor objects. This argument is
-            relevant only if ``auxiliary_dict_file`` is not set to ``False``.
+            List of monitor objects' names to export. The default is the names of all
+            monitor objects. This argument is relevant only if ``auxiliary_dict_file``
+            is not set to ``False``.
         datasets : list, optional
-            List of dataset names to export. The default is all datasets. This argument is relevant only if
-            ``auxiliary_dict_file`` is not set to ``False``.
+            List of dataset names to export. The default is all datasets. This argument
+             is relevant only if ``auxiliary_dict_file`` is not set to ``False``.
```

Comment:
```suggestion
             is relevant only if ``auxiliary_dict_file`` is set to ``True``.
```
Wouldn't it be better to phrase this without a "not set to" construction? It would be easier to understand.